arch/cpu/cc26x0-cc13x0/rf-core/smartrf_fxn.h

```diff
@@ -0,0 +1,14 @@
+
+#ifndef SMARTRF_FXN_H
+#define SMARTRF_FXN_H
+
+typedef struct {
+    int rfMode;
+    void (*cpePatchFxn)(void);
+    void (*mcePatchFxn)(void);
+    void (*rfePatchFxn)(void);
+
+} RF_Mode;
+
+#endif
+
```

Comment:
Is this file coming from SmartRF studio or is it from the SDK? Need to figure out the license on it

Reply:
I think it comes from the simplelink SDK 1.6